netlify.toml

```diff
@@ -1,3 +1,15 @@
 [build]
 command = "npm run docsite build"
 publish = "docsite-build"
+
+[[redirects]]
+  from = "/"
```

Comment:
@frangio I think it should be "/*"

Reply:
I'd prefer to keep this conservative redirect for now.